src/bundle/Resources/public/js/scripts/core/multilevel.popup.menu.js

```diff
@@ -200,6 +203,13 @@
         }
 
         closeBranch(branchElement) {
+            const searchInput = branchElement.querySelector('.ibexa-multilevel-popup-menu__search-input');
+
+            if (searchInput?.value !== '') {
+                searchInput.value = '';
+                searchInput.dispatchEvent(new Event('input'));
+            }
```

Comment:
I think this has to be placed inside `toggleBranch` otherwise, it will be inconsistent.